### PR TITLE
Adding controllers

### PR DESCRIPTION
**Pull request by nicolepeak**
Ok team. I did A LOT of changes on this branch and I'm so sorry for that. I know this is not the way things are supposed to be done.

a couple things to note:

- I added structure to the controllers and somewhat completed the UserController. 
- I added UserService in a new service layer
- I moved some of the logic from the LoginFormDTO to a LoginController
- I commented out shelly's authenticationFilter and tried something new underneath to check out
- changed some things in the User.Java model to try to streamline the user table in mySQL (it was showing the password in the table and we don't want that. It's still showing name and username so we need to fix that also). 

*I'm SO sorry this review is GIANT. I got carried away and will break it up in the future. One thing just kept leading to another.

### CODE REVIEW

File: The-Coding-Sharks-api/src/main/java/com/example/The_Coding_Sharks_demo/models/DTO/LoginFormDTO.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+
```
Comment (adc009) [inline The-Coding-Sharks-api/src/main/java/com/example/The_Coding_Sharks_demo/models/DTO/LoginFormDTO.java:37]: For clarity, could we remove this commented code?

File: The-Coding-Sharks-api/src/main/java/com/example/The_Coding_Sharks_demo/models/User.java
Comment (adc009) [inline The-Coding-Sharks-api/src/main/java/com/example/The_Coding_Sharks_demo/models/User.java:1]: Thanks for cleaning up the username / inherited name field! I think this way makes more sense than what I originally had, plus it seems better adapted for other pieces of code.

**Review by adc009 — APPROVED** [review]
Code looks good to me! I left a couple comments. Let's run through what the new code does when we get a chance, to make sure we're all on the same page.

File: The-Coding-Sharks-api/src/main/java/com/example/The_Coding_Sharks_demo/models/DTO/LoginFormDTO.java (hunk 1 above)
Comment (nicolepeak) [reply The-Coding-Sharks-api/src/main/java/com/example/The_Coding_Sharks_demo/models/DTO/LoginFormDTO.java:37]: absolutely. Wanted to make sure I remembered to tell Shelly about this. thanks for pointing this out. I will remove it.

**Pull request merged**